CHANGELOG.md

```diff
@@ -4,12 +4,16 @@ ChangeLog
 Unreleased
 -----------------
 ### Added
+* Added: `custom-property-no-duplicate-declaration` rule to disallow a custom property to be declared more than once with a different fallback value
+
+1.3.0 - (October 30, 2018)
```

Comment:
The change log wasn't automatically updated during the last release so I manually added an entry.